README.md

```diff
@@ -15,7 +62,7 @@ This README outlines the details of collaborating on this Ember addon.
 
 ## Running Tests
 
-* `npm test` (Runs `ember try:each` to test your addon against multiple Ember versions)
+* `yarn test` (Runs `ember try:each` to test your addon against multiple Ember versions)
 * `ember test`
 * `ember test --server`
 
```

Comment:
Maybe add which ember version this supports

Reply:
Good idea. I don't know which ember version is supported. Will make a TODO